src/borrow_tracker/tree_borrows/tree.rs

```diff
@@ -458,15 +464,18 @@ impl Tree {
                     tag: root_tag,
                     parent: None,
                     children: SmallVec::default(),
-                    default_initial_perm: root_perm,
+                    default_initial_perm: root_default_perm,
                     debug_info,
                 },
             );
             nodes
         };
         let rperms = {
             let mut perms = UniValMap::default();
-            perms.insert(root_idx, LocationState::new(root_perm).with_access());
+            // we manually set it to `Active` on all in-bounds positions.
+            // we also ensure that it is initalized, so that no `Active` but
+            // not yet initialized nodes exist.
```

Comment:
```suggestion
            // We manually set it to `Active` on all in-bounds positions.
            // We also ensure that it is initalized, so that no `Active` but
            // not yet initialized nodes exist. Essentially, we pretend there
            // was a write that initialized these to `Active`.
```

Reply:
Done

Reply:
Not dropped part of my diff, I had also fixed capitalization in the first line.

That's why using the github button is a good idea. ;)  (You can than still pull, squash, force-push if you want to avoid the extra commit.)

Reply:
Ah, alternatively one can also not be blind :see_no_evil: 
I've also fixed capitalization in other places.